apps/namada-interface/vite.config.js

```diff
@@ -0,0 +1,19 @@
+import react from "@vitejs/plugin-react";
+import { defineConfig } from "vite";
+import tsconfigPaths from "vite-tsconfig-paths";
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react(), tsconfigPaths()],
+  define: {
+    "process.env": {},
```

Comment:
Is this enough to load envs?

Reply:
It was a temp fix that I left behind :/ Checking that now